scripts/feature_toggle_report.py

```diff
@@ -43,12 +40,16 @@
     default=None,
     help='specify toggle types if you only want data on certain toggle type',
     )
+@click.option(
+    '--summarize', is_flag=True,
+    help="if you want a simplified version of report",
```

Comment:
Ideally, could you clarify here what a simplified version of the report is? If that is too complicated, could you add that to the docs somewhere?

Reply:
Before I make further changes, what do you think of summarize being default and creating a --verbose flag for the full report? I think that would make documenting easier

Reply:
See my other comment for what I care about most in terms of documentation.  If you want to call one of these the verbose version of the report, and switch defaults, that is fine.